src/js/components/Form/doc.js

```diff
@@ -45,6 +45,11 @@ export const doc = Form => {
       `Function that will be called when the form is reset. The
       single argument is the event provided by react.`,
     ),
+    onValidate: PropTypes.func.description(
+      `Function that will be called when the form is validated. The
+      single argument is an event containing the latest error object
+      via \`event.error\` and info object via \`event.info\`.`,
```

Comment:
I think these should be `event.errors` and `event.infos` to align with the property names.

Reply:
I think that's a good idea